### DISCUSSION TITLE
DEV - Code: Relative imports.

### DISCUSSION THREAD

**Discussion by real-yfprojects** [borgbase/vorta#1159] [Ideas]
I was looking at the code of *vorta* and noticed that absolute imports instead of relative imports are used for importing other modules of the `vorta` package. Is there a reason for that?
I could create a PR converting all the package-internal imports to relative ones.

##### Example 1

https://github.com/borgbase/vorta/blob/eadb3e709a032e750d22aa1e3db82528d1642022/src/vorta/__main__.py#L12
would be changed to
```python
from .utils import parse_args
```
##### Example 2 

https://github.com/borgbase/vorta/blob/eadb3e709a032e750d22aa1e3db82528d1642022/src/vorta/borg/check.py#L1
Already in place :shrug:

##### Example 3
https://github.com/borgbase/vorta/blob/eadb3e709a032e750d22aa1e3db82528d1642022/src/vorta/borg/prune.py#L2
would be converted to
```python
from ..utils import format_archive_name
```

**Comment by m3nu** [comment]
I intentionally use absolute imports unless it's the same package. This is also what's suggested in the[official docs](https://www.python.org/dev/peps/pep-0008/#imports).

> Absolute imports are recommended, as they are usually more readable...

I also find absolute imports easier to find and update, if a package is renamed.

Still appreciate the effort and concern to improve something, but changing `vorta.utils` to `..utils` might get us into bike-shedding territory. 😬 

We have more urgent issues open. Especially those marked [Good first issue](https://github.com/borgbase/vorta/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) are there to get started and get to know the code.

Thanks and happy new year!

**Reply by real-yfprojects** [reply]
I agree. After setting up borg and vorta I might have the time to look at one of the *good first issues*.
Happy new year to you too!
